packages/core/package.json

```diff
@@ -24,13 +24,14 @@
     "typescript-standard": "0.3.36"
   },
   "dependencies": {
-    "wot-typescript-definitions": "0.7.3",
     "@node-wot/td-tools": "0.7.7",
-    "uuid": "3.4.0",
+    "@petamoriken/float16": "^3.1.1",
+    "@types/uritemplate": "0.3.4",
```

Comment:
I don't know why but every time I install dependency things get shuffled. It seems my npm version doesn't like the current ordering... In this PR I added `@petamoriken/float16` for `float16` coding and deconding

Reply:
NPM seems to order them alphabetically. Do you know?

Maybe we should go once over all dependencies and sort them alphabetically so that we do not get these unnecessary diffs all the time. What do you think?

Reply:
let's discuss that in https://github.com/eclipse/thingweb.node-wot/issues/450